dapps/shop/backend/data/migrations/20200519035456-addShopDeployments.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(() => {
+      queryInterface.createTable('shop_deployments', {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        shop_id: {
+          type: Sequelize.INTEGER,
+          references: { model: 'shops', key: 'id' }
+        },
+        ipfs_hash: Sequelize.STRING,
+        created_at: Sequelize.DATE,
+        updated_at: Sequelize.DATE
```

Comment:
Does it make sense to include the configured DNS URL here as well?  I'm not sure if that can/would change from deployment to deployment.  It's currently displayed on that deployment confirm page.

Reply:
Yea I think DNS url makes sense to add. Perhaps also the IPFS gateway the hash is pinned to (eg pinata)

Reply:
Good idea. Added domain and ipfs_gateway to the shop_deployment table.